src/messages.js

```diff
@@ -39,6 +39,10 @@
         }
       }
     }
+    // Narrating every call to umsg
+    if($(this.jsav.container).attr("voice") === "true") {
+      JSAV.ext.textToSpeech(msg);
```

Comment:
This could be changed to `this.jsav.textToSpeech)msg);`. When things are added to JSAV.ext, they'll become instance functions on jsav objects.